### ISSUE TITLE
Keymap doesn't work

### ISSUE THREAD

**Issue by ryanberckmans** [amannn/atom-format-javascript-comment#7]
The default keymap doesn't work for me. Keymap resolver shows that it's correctly identifying the function, but it's not executed. Was unable to get a manual keymap working.

![image](https://user-images.githubusercontent.com/299324/33796971-2760cf92-dccd-11e7-8144-9bfb1b5c9d91.png)

The menu `cmd-shift-p` --> `Format Comment` works for me. For the same unformatted comment, I am able to make it work with the menu `Format Comment`, but unable to trigger a keymap.

After adding debug console msgs, it's clear the keymap is never triggering the package function.

I thought this might be related to the rename from `format-javascript-comment` -> `format-comment`. Tried renaming the entire package to `format-comment` and had no success with the keymap.

**Comment by amannn** [comment]
Hi @ryanberckmans!

Thanks for opening this issue and #8. I also noticed recently that the keybinding doesn't work properly.

My plan would be to eventually deprecate this plugin in favour of [supporting comment wrapping prettier](https://github.com/prettier/prettier/issues/265). It looks like there's not so much missing, as there's now markdown support in prettier and a `fill` layout primitive so this feature could be implemented by reusing those.

There are a couple more issues with the current implementation and I'd have to restructure the core a bit. I started that some time ago, but couldn't find the time to finish it.

Long story short, I think we should redirect our efforts towards the prettier integration.

**Comment by ryanberckmans** [comment]
@amannn sounds good! The current functionality, accessed with the "Format Comment" menu item, has been very useful for me over the past week.

**Comment by ryanberckmans** [comment]
Suggest that we leave this open since the package in current state seems to be useful.
